src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -220,7 +223,12 @@ message CloseSharedMemoryResourcesResponse {
   map<string, bool> close_map_results = 1;
 }
 
-// Host tells the worker to load a Function
+// Host tells the worker to load a list of Functions
+message FunctionLoadRequests {
```

Comment:
Should this be singular? `FunctionLoadRequest` ? The child property can be plural as it is a collection type.

Reply:
`FunctionLoadRequest` singular already exists and is used in the current implementation for sending each function load request one by one from host to worker. The definition is a few lines down in this file. 

In order to not break the current implementation but provide a path for improving performance, `FunctionLoadRequests` wraps the singular `FunctionLoadRequest` messages that are currently being sent one by one into one payload instead. Workers will have to opt into this using the capability @surgupta-msft added. 

Please let me know if I understand correctly @surgupta-msft ... this is also what I meant about the naming confusion.

Reply:
Agree, please make sure this is singular

Reply:
That makes sense, I misread the code :)

Reply:
@satvu yes your understanding is correct. I have updated the code as well to avoid naming confusion.